src/System.IO.FileSystem/ref/System.IO.FileSystem.cs

```diff
@@ -212,3 +224,77 @@ public enum SearchOption
         TopDirectoryOnly = 0,
     }
 }
+namespace System.IO.Enumeration
+{
+    public enum MatchType
+    {
+        Simple,
+        Dos
+    }
+    public class EnumerationOptions
+    {
+        public EnumerationOptions() { }
+
+        public bool RecurseSubdirectories { get { throw null; } set { } }
+        public bool IgnoreInaccessible { get { throw null; } set { } }
+        public int BufferSize { get { throw null; } set { } }
+        public FileAttributes AttributesToSkip { get { throw null; } set { } }
+        public MatchType MatchType { get { throw null; } set { } }
+        public bool ReturnSpecialDirectories { get { throw null; } set { } }
+    }
+    public ref struct FileSystemEntry
+    {
+        public ReadOnlySpan<char> Directory { get { throw null; } }
+        public string RootDirectory { get { throw null; } }
+        public string OriginalRootDirectory { get { throw null; } }
+        public ReadOnlySpan<char> FileName { get { throw null; } }
+        public FileAttributes Attributes { get { throw null; } }
+        public long Length { get { throw null; } }
+        public DateTimeOffset CreationTimeUtc { get { throw null; } }
+        public DateTimeOffset LastAccessTimeUtc { get { throw null; } }
+        public DateTimeOffset LastWriteTimeUtc { get { throw null; } }
+
+        public bool IsNameDotOrDotDot { get { throw null; } }
```

Comment:
IsNameDotOrDotDot and ReturnSpecialDirectories seem to be two names for the same thing. Can we make them consistent? Eg., either IsSpecialDirectory or ReturnDotAndDotDot

Reply:
They aren't strictly the same. One actually only applies to directories, the other applies to any entry- although I'm unaware of the ability to actually name a file that way. Definitely cannot in Windows, but I'm not positive elsewhere.

Reply:
With further reflection I realize having the helper is unnecessary now that we have an option and returning them is turned off by default. Removing the property.